runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationPlugin.java

```diff
@@ -54,11 +63,22 @@ ItemIdentificationConfig getConfig(ConfigManager configManager)
 	protected void startUp()
 	{
 		overlayManager.add(overlay);
+		client.setOverrideMembersItemNames(config.membersObject());
```

Comment:
this call is not threadsafe

Reply:
There's still a call to this in `startUp`, which runs on EDT, not on client thread.

Reply:
Whoops, can't believe I forgot that. Fixed it.